scripts/README.md

```diff
@@ -0,0 +1,25 @@
+# Publish and Tag Docker Image
+
+The `publish-docker.sh` script does the following interactively:
+
+- Ask which packages to  include (always answer by SPACE-RETURN)
```

Comment:
I think it would be easier to just include all packages. It is one less thing to figure out to do the release and it means that I can always pick all the packages with the same version and they are guaranteed to work together

Reply:
All the packages mentioned in the config.json will always go together, so whichever package you choose here does not really metter.